build.gradle

```diff
@@ -45,6 +45,7 @@ task detektAll(type: io.gitlab.arturbosch.detekt.Detekt) {
     exclude("**/test/**/*.kt")
     exclude("**/testDefaults/**/*.kt")
     exclude("**/testCustomEntitlementComputation/**/*.kt")
+    exclude("**/CustomEntitlementComputationSample/**/*.kt")
```

Comment:
We probably would want to run linter on the sample app. But we can do that in a followup PR